lib/passport-wsfed-saml2/saml.js

```diff
@@ -373,6 +373,10 @@ SAML.prototype.parseAssertion = function(samlAssertion, callback) {
     return callback(new Error('SPNameQualifier attribute in the NameID element does not match ' + self.options.realm), null);
   }
 
+  if (self.options.issuer && self.options.issuer !== issuer) {
```

Comment:
should this be case insensitive? (not saying it should, but we should check the protocol)

Reply:
@donflopez has confirmed this was resolved outside GH